types/vue3-carousel-3d/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for vue3-carousel-3d 1.0
+// Project: https://github.com/Ray93/vue3-carousel-3d
+// Definitions by: tevem1207 <https://github.com/tevem1207>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.9
```

Comment:
```suggestion
// TypeScript Version: 4.4
```
This can be 4.4 for index signature access, why 4.9?

Reply:
It was because the version I use is 4.9. I changed it to 4.4 according to your opinion. thx :)